packages/nimble-components/src/patterns/tabs/template.ts

```diff
@@ -0,0 +1,67 @@
+import { html, ref, slotted, when } from '@microsoft/fast-element';
+import type { ViewTemplate } from '@microsoft/fast-element';
+import {
+    endSlotTemplate,
+    startSlotTemplate,
+    FoundationElementTemplate,
+    TabsOptions
+} from '@microsoft/fast-foundation';
+import type { Tabs } from '../../tabs';
+import type { AnchorTabs } from '../../anchor-tabs';
+import { buttonTag } from '../../button';
+import { iconArrowExpanderLeftTag } from '../../icons/arrow-expander-left';
+import { iconArrowExpanderRightTag } from '../../icons/arrow-expander-right';
+
+type TabsOrAnchorTabs = Tabs | AnchorTabs;
+
+// prettier-ignore
+export const template: FoundationElementTemplate<
+ViewTemplate<TabsOrAnchorTabs>,
+TabsOptions
+> = (context, definition) => html`
+    <div
+        class="tab-bar"
+    >
+        ${startSlotTemplate(context, definition)}
+        ${when(x => x.showScrollButtons, html<Tabs>`
+            <${buttonTag} 
+                content-hidden
+                class="scroll-button left"
+                appearance="ghost"
```

Comment:
You can bind to the value of "ghost" here instead of hard-coding the string: `appearance="${ButtonAppearance.ghost}"`. Same for the scroll right button. Then